### ISSUE TITLE
Time Seeded Column (Enhancement)

### ISSUE THREAD

**Issue by Sitruk** [omaralvarez/deluge-autoremoveplus#30]
Would it be possible to incorporate a column that shows the amount of time a torrent has been seeding? The Seedtime plugin has one and I found it useful (https://github.com/gazpachoking/deluge-seedtime)

**Comment by omaralvarez** [comment]
Yes, it is certainly possible. Although I don't know when I will have time to do it. Onto the backlog it is.
